### PR TITLE
S/fixes and suggestions

### PR DESCRIPTION
**Pull request by stevenweber**

### CODE REVIEW

**Comment by ebenoist** [comment]
> primary
?

I thought the new default was main? Or is that how they have their repo setup?

**Comment by stevenweber** [comment]
> > primary
> > ?
> 
> I thought the new default was main? Or is that how they have their repo setup?

That's how the source repo was configured https://github.com/muenzpraeger/eslint-plugin-inclusive-language/branches

File: lib/rules/use-inclusive-words.js [hunk 1]
```diff
@@ -7,12 +7,16 @@ const {
...
+
 const DEFAULT_MESSAGE =
     "The usage of the non-inclusive word '{{word}}' is discouraged, use '{{suggestion}}' instead.";
+const PROJECT_URL =
+    'http://github.com/muenzpraeger/eslint-plugin-inclusive-language/tree/primary/docs/rules/use-inclusive-words.md';
 const RULE_CATEGORY = 'Language';
 const RULE_DESCRIPTION = 'highlights where non-inclusive language is used';
 // eslint-disable-next-line no-unused-vars
-const SUGGESTION_MESSAGE = "Replace word '{{word}}'' with '{{suggestion}}.'"; // Currently not in use
+const SUGGESTION_MESSAGE = "Replace word '{{word}}' with '{{suggestion}}.'"; // Currently not in use
```
Comment (ebenoist) [inline lib/rules/use-inclusive-words.js]: Is this comment still valid?

File: lib/rules/use-inclusive-words.js [hunk 2]
```diff
@@ -42,48 +46,69 @@ function excludeAllowedTerms(words) {
...
 
-    if (!result || result.length == 0) return;
+    if (!result) return;
 
-    const data = {
-        word: result[0].word,
-        suggestion: result[0].suggestion
-    };
+    const { word, explanation } = result;
+    // backwards-compatibility with singular suggestions
```
Comment (ebenoist) [inline lib/rules/use-inclusive-words.js:84]: fancy

File: lib/rules/use-inclusive-words.js [hunk 3]
```diff
@@ -117,6 +142,9 @@ module.exports = {
             Identifier(node) {
                 validateIfInclusive(context, node, node.name);
             },
+            JSXIdentifier(node) {
```
Comment (ebenoist) [inline lib/rules/use-inclusive-words.js:144]: 👍

**Comment by ebenoist** [comment]
Nice work!

**Review by ebenoist — APPROVED** [review]

File: lib/rules/use-inclusive-words.js (hunk 1 above)
Comment (stevenweber) [reply lib/rules/use-inclusive-words.js]: It's not!

**Pull request merged**